openquake/calculators/event_based.py

```diff
@@ -428,6 +428,46 @@ def compute_avg_gmf(gmf_df, weights, min_iml):
     return dic
 
 
+def read_gsim_lt(oq):
+    # in aristotle mode the gsim_lt is read from the exposure.hdf5 file
+    gsim_lt = readinput.get_gsim_lt(oq)
+    if oq.aristotle:
+        if not oq.mosaic_model:
+            if oq.rupture_dict:
+                lon, lat = [oq.rupture_dict['lon'], oq.rupture_dict['lat']]
+            elif oq.rupture_xml:
+                hypo = readinput.get_rupture(oq).hypocenter
+                lon, lat = [hypo.x, hypo.y]
+            mosaic_models = get_close_mosaic_models(lon, lat, 5)
+            # NOTE: using the first mosaic model
+            oq.mosaic_model = mosaic_models[0]
+            if len(mosaic_models) > 1:
+                logging.info('Using the "%s" model' % oq.mosaic_model)
+        [expo_hdf5] = oq.inputs['exposure']
+        if oq.mosaic_model == '???':
+            raise ValueError(
+                '(%(lon)s, %(lat)s) is not covered by the mosaic!' %
+                oq.rupture_dict)
+        if oq.gsim != '[FromFile]':
+            raise ValueError(
+                'In Aristotle mode the gsim can not be specified in'
+                ' the job.ini: %s' % oq.gsim)
+        if oq.tectonic_region_type == '*':
+            raise ValueError(
+                'The tectonic_region_type parameter must be specified')
+        gsim_lt = logictree.GsimLogicTree.from_hdf5(
+            expo_hdf5, oq.mosaic_model,
+            oq.tectonic_region_type.encode('utf8'))
+        # add with_betw_ratio when only the total stddev is defined
+        betw = {'with_betw_ratio': 1.7}
```

Comment:
I think we should try to avoid the hardcoding of the value for this parameter, especially because the ratio might be different for different tectonic region types. Can we surface this through the interface in some way so that the engine doesn't make an implicit decision on behalf of the user?